dvc/command/plots.py

```diff
@@ -123,7 +124,12 @@ def add_parser(subparsers, parent_parser):
         "--template",
         nargs="?",
         default=None,
-        help="File to be injected with data.",
+        help=(
+            "Vega specification file to inject with the data. "
```

Comment:
According to https://dvc.org/doc/command-reference/plots#plot-templates a template is a DVC specification file. It's not a Vega standard right? Maybe just say
```suggestion
            "Plot template file to inject with the data. "
```
Or am I not getting what plot templates are?

Same in line 192

Reply:
I found this https://github.com/vega/schema are templates basically this? Vega schema files. If so, I suggest
```suggestion
            "Vega JSON schema file to inject with the data. "
```
and will update the docs to link to this repo for reference.

Reply:
Hmm actually template is an `Vega JSON schema` with DVC placeholders (anchors) that will be filled with proper values after using `dvc plots`. Also, template does not have to be JSON at all,
that is bad on my side. If user wants it can be HTML as well, the only thing that makes it template is existence of `DVC_METRIC_DATA` anchor inside. 

Also this might change depending on discussion under https://github.com/iterative/dvc/issues/3818

Reply:
OK np let's just come up with a general description here and we'll have to revisit the docs to explain this a little better. The anchors are listed but it's not clear that templates are a combo of Vega schema and DVC anchors + possibly HTML. TBH I still don't totally get it haha. But how about

```suggestion
            "Special JSON or HTML schema to inject with the data. "
```
?

> Also this might change depending on discussion under #3818

No worries. We can revisit this and docs if/when that happens.

Reply:
p.s. added checkbox to improve plot template desc. to #1175.